rotkehlchen/chain/ethereum/modules/convex/constants.py

```diff
@@ -0,0 +1,194 @@
+from rotkehlchen.chain.ethereum.types import string_to_ethereum_address
+
+
+CPT_CONVEX = 'convex'
+
+BOOSTER = string_to_ethereum_address('0xF403C135812408BFbE8713b5A23a04b3D48AAE31')
+CVX_LOCKER = string_to_ethereum_address('0xD18140b4B819b895A3dba5442F959fA44994AF50')
+
+CVX_WITHDRAW_TOPIC = b'p\x84\xf5Gf\x18\xd8\xe6\x0b\x11\xef\r}?\x06\x91FU\xad\xb8y>(\xff\x7f\x01\x8dLv\xd5\x05\xd5'  # noqa: E501
+CVXCRV_REWARD_PAID_TOPIC = b'\xe2@6@\xbah\xfe\xd3\xa2\xf8\x8buWU\x1d\x19\x93\xf8K\x99\xbb\x10\xff\x83?\x0c\xf8\xdb\x0c^\x04\x86'  # noqa: E501
+CVXCRV_WITHDRAWAL_TOPIC = b'p\x84\xf5Gf\x18\xd8\xe6\x0b\x11\xef\r}?\x06\x91FU\xad\xb8y>(\xff\x7f\x01\x8dLv\xd5\x05\xd5'  # noqa: E501
+TRANSFER_TOPIC = b'\xdd\xf2R\xad\x1b\xe2\xc8\x9bi\xc2\xb0h\xfc7\x8d\xaa\x95+\xa7\xf1c\xc4\xa1\x16(\xf5ZM\xf5#\xb3\xef'  # noqa: E501
+CVX_LOCKER_REWARD_PAID_TOPIC = b'T\x07\x98\xdfF\x8d{#\xd1\x1f\x15o\xdb\x95L\xb1\x9a\xd4\x14\xd1Pr*{mU\xba6\x9d\xeay.'  # noqa: E501
+BOOSTER_WITHDRAW_TOPIC = b'\x92\xcc\xf4P\xa2\x86\xa9W\xafRP\x9b\xc1\xc9\x93\x9d\x1ajH\x17\x83\xe1B\xe4\x1e$\x99\xf0\xbbf\xeb\xc6'  # noqa: E501
+
+WITHDRAWAL_TOPICS = {
+    CVXCRV_WITHDRAWAL_TOPIC,
+    CVX_WITHDRAW_TOPIC,
+    BOOSTER_WITHDRAW_TOPIC,
+}
+
+REWARD_TOPICS = {
+    CVXCRV_REWARD_PAID_TOPIC,
+    CVX_LOCKER_REWARD_PAID_TOPIC,
+}
+
+CONVEX_ABRAS_HEX = {  # using set since we only need to check if an address is "in" these pools
+    b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xd9$\x94\xcb\x92\x1e\\\r:9\xea\x88\xd0\x14{\xbd\x82\xe5\x10\x08',  # noqa: E501
+    b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00YX\xa8\xdb}\xfe\x0c\xc4\x93\x82 \x90i\xb0\x0fT\xe1y)\xc2',  # noqa: E501
+    b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xb6^\xde\x13E!\xf0\xef\xd4\xe9C\xc85\xf4P\x13}\xc6\xe8>',  # noqa: E501
+    b"\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00;\xa2\x07\xc2Z'\x85$\xe1\xcc\x7f\xaa\xea\x95\x07S\x04\x90r\xa4",  # noqa: E501
+}
+
+CONVEX_BALANCE_ABI = [{"inputs": [{"internalType": "address", "name": "account", "type": "address"}], "name": "balanceOf", "outputs": [{"internalType": "uint256", "name": "", "type": "uint256"}], "stateMutability": "view", "type": "function"}]  # noqa: E501
+CONVEX_POOLS = {  # pool address: reward token address
+    string_to_ethereum_address('0xf34DFF761145FF0B05e917811d488B441F33a968'): string_to_ethereum_address('0x845838DF265Dcd2c412A1Dc9e959c7d08537f8a2'),  # noqa: E501
```

Comment:
Do you have any ideas on how to avoid it?

Reply:
I have an idea!

It may sound like an overkill but since you take them from https://convexfinance.com/stake you can actually write a unit test that uses beautifulsoup (we already have it as a dependency) to scrape the site and compare the list in the site to this list. Beautifulsoup is really easy to use and fun! It's "the" python scraping library.

Heck the code could also be abstracted to auto-fill the missing stuff in the list but perhaps that would be overengineering! :)